src/components/popup-menu/popup-menu.scss

```diff
@@ -0,0 +1,38 @@
+$userPanelBackground: $blueSecondary;
+$userPanelPadding: $layoutDefaultPadding;
+
+$include-html: false !default;
+
+@if ($include-html) {
+  .mint-popup-menu {
+    position: relative;
+    width: 100%;
+    padding: $userPanelPadding/3 $userPanelPadding/2;
```

Comment:
We have it discussed here (and made a decision): https://github.com/brainly/style-guide/issues/244

Is the decision there incorrect or not full? WDYT @vergilius @kdzwinel